make-coe/make-coe.go

```diff
@@ -20,6 +20,27 @@ type MakeCOE struct {
 	Account          *Account
 }
 
+func handleNotAcceptable(err libcarina.HTTPErr) error {
+	return errors.Wrap(err, "Unable to communicate with the Carina API because the client is out-of-date. Update the carina client to the latest version. See https://getcarina.com/docs/tutorials/carina-cli#update for instructions.")
+}
+
+func handleHTTPError(err libcarina.HTTPErr) error {
+	switch err.StatusCode {
+	default:
+		return err
+	case 406:
+		return handleNotAcceptable(err)
+	}
+}
+
+func handleLibcarinaError(err error) error {
```

Comment:
No worries. I will narrow the scope if it gets hairy, but otherwise, it might not take me much longer to do the whole surface.